leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -40,22 +40,30 @@
  */
 public class RedisSecurityStore implements EditableSecurityStore {
 
-    private static final String SEC_EP = "SEC#EP#";
+    private final String secEp;
 
-    private static final String PSKID_SEC = "PSKID#SEC";
+    private final String pskIdSec;
```

Comment:
I don't know if you agree with me but I feel this attribute names are not so clear.
Maybe good opportunity to change it to something clearer ?

`secEp` => `securityInfoByEndpointPrefix`
`pskIdSec` => `endpointByIdentityKey`

If we change this we need to also change it for constructor arguments at : 
`public RedisSecurityStore(Pool<Jedis> pool, String secEp, String pskIdSec)`

Reply:
Totally agree, more descriptive names will be better.